### PR TITLE
Bug 1880302: Bump K8s dependencies to v1.19.0

### PR DESCRIPTION
**Pull request by enxebre**
Our dependencies are out of date and should be on K8s 1.19.0 before the 4.6 release to match the rest of the core components. This PR updates all of the K8s and openshift depdendencies to include the lastest 1.19.0 release.
This address https://bugzilla.redhat.com/show_bug.cgi?id=1880302 and https://bugzilla.redhat.com/show_bug.cgi?id=1877743

### CODE REVIEW

**Comment by p0lyn0mial** [comment]
@enxebre https://bugzilla.redhat.com/show_bug.cgi?id=1880302 targets release 4.5

**Comment by enxebre** [comment]
>@enxebre https://bugzilla.redhat.com/show_bug.cgi?id=1880302 targets release 4.5

We need a ticket targeting 4.6 even though this was first found in 4.5 https://bugzilla.redhat.com/show_bug.cgi?id=1880302.
The clone targeting 4.5.z is here https://bugzilla.redhat.com/show_bug.cgi?id=1880320

**Review by elmiko — APPROVED** [review]
/approve

File: go.mod [hunk 1]
```diff
@@ -5,19 +5,20 @@ go 1.13
...
-	k8s.io/apimachinery v0.18.0
+	github.com/go-logr/logr v0.2.1 // indirect
+	github.com/googleapis/gnostic v0.5.1 // indirect
+	github.com/openshift/api v0.0.0-20200916161728-83f0cb093902
+	github.com/openshift/client-go v0.0.0-20200827190008-3062137373b5
+	github.com/openshift/library-go v0.0.0-20200917093739-70fa806b210a
+	github.com/stretchr/testify v1.5.1
+	k8s.io/api v0.19.0
+	k8s.io/apimachinery v0.19.0
 	k8s.io/client-go v12.0.0+incompatible
```
Comment (michaelgugino) [inline go.mod:16]: Should we update this dependency?  It's very old and they have changed how the release versions on client-go.

Comment (enxebre) [reply go.mod:16]: the one applying is this one https://github.com/openshift/cluster-autoscaler-operator/pull/166/files#diff-37aff102a57d3d7b797f152915a6dc16L23 v0.19.0

**Review by alexander-demichev — APPROVED** [review]
/lgtm

**Pull request merged**